applications/asset_tracker_v2/overlay-debug.conf

```diff
@@ -50,5 +50,9 @@ CONFIG_AWS_IOT_LOG_LEVEL_DBG=y
 CONFIG_AWS_FOTA_LOG_LEVEL_DBG=y
 CONFIG_AWS_JOBS_LOG_LEVEL_DBG=y
 
+# Azure IoT Hub
+CONFIG_AZURE_IOT_HUB_LOG_LEVEL_DBG=y
+CONFIG_AZURE_FOTA_LOG_LEVEL_DBG=y
```

Comment:
```suggestion
# CONFIG_AZURE_IOT_HUB_LOG_LEVEL_DBG=y
# CONFIG_AZURE_FOTA_LOG_LEVEL_DBG=y
```

Should be disabled by default, otherwise I get these warnings:

```
warning: The choice symbol AZURE_FOTA_LOG_LEVEL_DBG (defined at
subsys/logging/Kconfig.template.log_config:20) was selected (set =y), but no symbol ended up as the
choice selection. See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_AZURE_FOTA_LOG_LEVEL_DBG.html and/or
look up AZURE_FOTA_LOG_LEVEL_DBG in the menuconfig/guiconfig interface. The Application Development
Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual
might be helpful too.


warning: The choice symbol AZURE_IOT_HUB_LOG_LEVEL_DBG (defined at
subsys/logging/Kconfig.template.log_config:20) was selected (set =y), but no symbol ended up as the
Parsing /workdir/ncs/firmware/Kconfig
choice selection. See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_AZURE_IOT_HUB_LOG_LEVEL_DBG.html
and/or look up AZURE_IOT_HUB_LOG_LEVEL_DBG in the menuconfig/guiconfig interface. The Application
Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of
the manual might be helpful too.
```